### PR TITLE
💅  / 🏗 Create new binary for compiler #compiler

### PR DESCRIPTION
**Pull request by samouri**
**summary**
Creates a new binary for the Bento Compiler project.
The first iteration acts as an identity function.

The purpose of this PR is to get bikeshedding out of the way re. the file location, name etc.

**binary contents**
```js
;
(function(){"use strict";function compileHtml(a){return a}function compileAst(a){return a}globalThis.compileHtml=compileHtml;globalThis.compileAst=compileAst})();//# sourceMappingURL=compiler.js.map
```

### CODE REVIEW

File: build-system/compile/bundles.config.js [hunk 1]
```diff
@@ -112,6 +112,15 @@ exports.jsBundles = {
       includePolyfills: true,
     },
   },
+  'compiler.js': {
+    srcDir: './src/compiler/',
```
Comment (rcebulko) [inline build-system/compile/bundles.config.js:116]: 🚲 🏚️ ing time! Do these binaries get served on the CDN?

**Review by rcebulko — COMMENTED** [review]
We should definitely add rules banning anything from importing from here, and enable type-checking for this directory right from the start

File: build-system/compile/bundles.config.js (hunk 1 above)
Comment (samouri) [reply build-system/compile/bundles.config.js:116]: I believe so!

**Comment by samouri** [comment]
> We should definitely add rules banning anything from importing from here, and enable type-checking for this directory right from the start


Done!

**Review by rcebulko — APPROVED** [review]

**Review by jridgewell — APPROVED** [review]

**Pull request merged**